run-tests.sh

```diff
@@ -23,9 +23,8 @@
 # waive the privileges and immunities granted to it by virtue of its status
```

Comment:
2017?

Reply:
I will update it in the https://github.com/inveniosoftware/invenio-search/pull/111 (I don't want to restart travis here :P )